assets/registry/replaces.template

```diff
@@ -1,11 +1,46 @@
 REGISTRY_QUERY_FILE="{{.OutputDir}}/registry.json"
 
-oc delete pod --ignore-not-found=true -n {{.Namespace}} grpcurl-query
+oc delete job --ignore-not-found=true -n {{.Namespace}} grpcurl-query
+
+set -e
+cat <<EOF | oc apply -n {{.Namespace}} --wait -f -
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    run: grpcurl-query
+  name: grpcurl-query
+spec:
+  template:
+    spec:
+      securityContext:
+        runAsUser: 1000910001
```

Comment:
```
Error creating: pods "grpcurl-query-" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, spec.containers[0].securityContext.runAsUser: Invalid value: 1000910001: must be in the ranges: [1000950000, 1000959999], provider "restricted": Forbidden: not usable by user or serviceaccount, provider "nonroot-v2": Forbidden: not usable by user or serviceaccount, provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "pcap-dedicated-admins": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler": Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2": Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, provider "splunkforwarder": Forbidden: not usable by user or serviceaccount, provider "node-exporter": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount]
```

Reply:
Yeah, i'm currently running the full test suite and realized that it wasn't as simple as i thought it would be, as there seem to be some differences in the SCCs between 4.11 and 4.12.

Reply:
@jbpratt Full test suite is now working against 4.11. Turns out you just need to omit the uid to make it work :D